backend/src/services/itinerary_generator.py

```
@@ -160,20 +171,25 @@ def _infer_destination(self, flights: List[Dict], hotels: List[Dict]) -> str:
-                        print(f"[DESTINATION] ✅ Mapped airport '{arrival_airport}' to destination: '{destination}'")
+                        print(f"[DESTINATION] ✅ Mapped airport "
+                              f"'{arrival_airport}' to destination: "
+                              f"'{destination}'")
                         return destination
                     else:
-                        print(f"[DESTINATION] ❓ Airport '{arrival_airport}' not in mapping")
+                        print(f"[DESTINATION] ❓ Airport '{arrival_airport}' "
+                              f"not in mapping")
 
-        print(f"[DESTINATION] ⚠️  Could not determine destination, using 'Unknown Destination'")
+        print(f"[DESTINATION] ⚠️  Could not determine destination, "
```

Ruff (0.13.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)